### ISSUE TITLE
古い発言やファイルの編集/削除に自動で追従できない

### ISSUE THREAD

**Issue by thinca** [vim-jp/slacklog#26]
今の方式だと、過去1日分より前の発言や添付テキストファイルの内容が編集・削除された場合に、追従することができません。
真面目にやろうと思ったら RTM API とかを使って常に監視する必要がありそうです。

恐らく優先度の面から、すぐに対応する必要はないと思いますが、将来へのメモとして起票しておきます。

**Comment by tyru** [comment]
確かに JSON エクスポート方式なら常に上書きですが API でポーリングする方式だと古い発言が消されても全部は見きれませんね。
RTM API があるならそれで監視するに関してもその通りだと思います 👍

**Comment by tyru** [comment]
> 今の方式だと、

（thinca さんは分かってるはずですが）
ちなみに現状は試験的にエクスポートした JSON で変換・表示だけしている状態で、新たな発言を取得する処理はまだありません。
